starfish/core/segmentation_mask.py

```diff
@@ -154,6 +170,38 @@ def from_label_image(
 
         return cls(masks)
 
+    def to_label_image(
+            self,
+            shape: Optional[Tuple[int]] = None,
+            *,
+            ordering: List[Axes] = [Axes.ZPLANE, Axes.Y, Axes.X]
```

Comment:
```suggestion
            ordering: Sequence[Axes] = (Axes.ZPLANE, Axes.Y, Axes.X)
```

rationale: default parameter values should be immutable, as they are only created once. if the method modifies `ordering`, then on the subsequent run, `ordering` will have the wrong default value.

```
In [1]: def method(default_param=[1, 2, 3]):
   ...:     default_param.append(4)
   ...:     print(default_param)
   ...:     

In [2]: method()
[1, 2, 3, 4]

In [3]: method()
[1, 2, 3, 4, 4]

In [4]: 
```

Reply:
this method doesn't mutate the default argument but I suppose it's better practice so I'll do it